internal/cli_utils/datasource_flag.go

```diff
@@ -0,0 +1,31 @@
+package cli_utils
+
+import (
+	"flag"
+	"fmt"
+	"slices"
+)
+
+type datasource_flag struct {
```

Comment:
type name should follow go conventions : DatasourceFlag

Reply:
Done